Sources/ToastWindow.swift

```diff
@@ -51,10 +51,10 @@ open class ToastWindow: UIWindow {
     let didBecomeActiveName = UIApplication.didBecomeActiveNotification
     #else
     self.windowLevel = .greatestFiniteMagnitude
-    let didBecomeVisibleName = NSNotification.Name.UIWindowDidBecomeVisible
-    let willChangeStatusBarOrientationName = NSNotification.Name.UIApplicationWillChangeStatusBarOrientation
-    let didChangeStatusBarOrientationName = NSNotification.Name.UIApplicationDidChangeStatusBarOrientation
-    let didBecomeActiveName = NSNotification.Name.UIApplicationDidBecomeActive
+    let didBecomeVisibleName = UIWindow.didBecomeVisibleNotification
```

Comment:
this change is incorrect, as we're in the #else for older Swift version: don't blindly trust the Migration assistant